packages/E2ETest/windows/ReactUWPTestApp/ReactUWPTestApp.csproj

```diff
@@ -132,7 +132,12 @@
     <None Include="ReactUWPTestApp_TemporaryKey.pfx" />
   </ItemGroup>
   <ItemGroup>
-    <Content Include="Assets\TreeDump\*.txt" />
+    <Content Include="Assets\TreeDump\ControlStyleRegularBorder.txt" />
+    <Content Include="Assets\TreeDump\ControlStyleRoundBorder.txt" />
+    <Content Include="Assets\TreeDump\ImageRTL.txt" />
+    <Content Include="Assets\TreeDump\ImageWithBorder.txt" />
+    <Content Include="Assets\TreeDump\ImageWithoutBorder-Subsequent.txt" />
+    <Content Include="Assets\TreeDump\ImageWithoutBorder.txt" />
```

Comment:
Ouch - this won't scale very well.  Why can't we use the wildcard here?

Reply:
@kmelmon asterisk is valid in msbuild but VS2019 chokes on it (says item is unparented when trying to do anything with one of the files)